_posts/chall12/2017-05-22-caroline-turtle-hack.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+author: chall12
+title: "Caroline's Turtle Exercise"
+---
+
+
+Here's the program I'm embedding:
+
+
+<iframe src="https://trinket.io/embed/python/6c004e7fe9" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+
+Reflection:
+
+My idea for making a flower began on the first day of class when we learned to make circles. I first changed the background 
+color to light blue and created a rectangle along the base of the screen filled green to make grass. To begin the flower, I clumped two 
+circles together for each petal (it took me a while to get the size of the circle and their distances to work so that it didn't look 
+like a mass of yellow). Then I had to find the center of the flower to make the pollen cluster. Afterwords, I calculated the distance
+between the grass and the base of the flower to create the stem. It would have been smarter to create the stem first and draw the flower 
```

Comment:
Nice workaround!  Doing things the hard way is a great way to learn.  Both the hard way and how to be more efficient the next time.